commons/commons-lang/src/main/java/com/clicktravel/common/security/ConfidentialString.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2014 Click Travel Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ */
+package com.clicktravel.common.security;
+
+/**
+ * Holds a {@link String} that has confidential information which should not be casually displayed. {@link #toString()}
+ * is defined to not include the confidential information.
+ */
+public class ConfidentialString {
```

Comment:
No, the contained String does that. Note that java.lang.String is a final class, so ConfidentialString contains a String rather than extending String. I did not want to delegate String methods as this would be make this class rather large and thus obscure its purpose.